tests/tools/redirect_logger_to_cout.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <spdlog/sinks/ostream_sink.h>
+#include <wmtk/utils/Logger.hpp>
+
+/**
+ * @brief setting default and wmtk logger to std::cout
+ *
+ * This is just a helper for debugging tests.
+ */
+inline void redirect_logger_to_cout()
+{
+    auto ostream_sink = std::make_shared<spdlog::sinks::ostream_sink_st>(std::cout);
+    auto ostream_logger = std::make_shared<spdlog::logger>("catch2_logger", ostream_sink);
+    ostream_logger->set_level(spdlog::level::debug);
+    spdlog::set_default_logger(ostream_logger);
```

Comment:
That's a Windows test environment thing. I added this function just for debugging the tests. It should never be used anywhere. Most probably that can also be solved differently, but I could not figure out how so far.

Reply:
What's the windows test environment?

Reply:
It's about a combination of spdlog, catch2, and Visual Studio. I followed the answer of [stackoverflow this thread](https://stackoverflow.com/questions/66473052/how-can-i-read-spdlog-output-in-a-google-test). I am most probably the only one who is using this but it makes my life definitely easier. I could also keep it as an external file on my computer only but I thought it's nice to have it available in case someone else has a similar issue.

Reply:
lol fair enough. just comment referring to this annoyance / linking to the stackoverflow